conf/application.conf

```diff
@@ -374,5 +374,11 @@ app.instanceRegistryUri = ${?INSTANCE_REGISTRY_URI}
 app.instanceRegistryBasePath = "localhost:8087"
 app.instanceRegistryBasePath = ${?INSTANCE_REGISTRY_BASE_PATH}
 
+//JwtKey
+play.http.secret.JWTkey="changeme"
+play.http.user="admin"
```

Comment:
remember to remove user and pass variable here, because they will be send by each client individually and there is no username / password authentication from the server itself